hw21cache/src/main/java/ru/otus/homework/cache/Cache.java

```diff
@@ -0,0 +1,27 @@
+package ru.otus.homework.cache;
+
+import java.util.function.Function;
+
+public interface Cache<K, V> {
+
+    //Добавить в кеш пару ключ/значение
```

Comment:
Лучше java doc 

```suggestion
/** Добавить в кеш пару ключ/значение */
```

в других местах тоже